realm/realm-library/src/objectServer/java/io/realm/SyncSession.java

```diff
@@ -115,6 +136,105 @@ void notifySessionError(int errorCode, String errorMessage) {
         }
     }
 
+    // Called from native code
```

Comment:
Native code is calling `SyncManager#notifyProgressListener()` instead of this.
Shouldn't we move this comment and following `@KeepMember` annotation to `SyncManager#notifyProgressListener()`?